src/fragment-components/a-code-snippet.tsx

```diff
@@ -98,17 +112,20 @@ export const componentInfo: ComponentInfo = {
 	defaultComponentObj: {
 		type: 'code-snippet',
 		variant: 'single',
-		code: ''
+		code: '',
+		light: false
```

Comment:
```suggestion
		code: ''
```

let's allow the default by default.